content/configuration/configuration_reference.md

```diff
@@ -1759,11 +1757,11 @@ is your task ID. Your configuration would be:
 If you check in some code with a commit message which includes the characters 'evo-512' then that will appear in the modification pop-up
 box as a link. When you click it, Go will take you to the web page `http://your-trackingtool/yourproject/512`.
 
-For example: If you use Mingle for your task manager, the configuration would be:
+For example: If you use Jira for your task manager, the configuration would be:
 
 ```xml
 <pipeline name="yourproject">
-  <trackingtool link="http://your-mingle-server/projects/yourproject/cards/${ID}" regex="##(\d+)"/>
+  <trackingtool link="http://your-jira-server/projects/yourproject/cards/${ID}" regex="##(\d+)"/>
```

Comment:
The JIRA URL style is very different. They use a `${PROJECT_ID}-${ISSUE_ID}` style template. https://issues.sonatype.org/browse/NEXUS-20838. Any examples using jira will need to reflect that.

Reply:
Done